test/hotspot/jtreg/compiler/compilercontrol/share/MultiCommand.java

```diff
@@ -78,12 +80,13 @@ public static AbstractTestBase generateRandomTest(boolean validOnly) {
                 md = AbstractTestBase.getValidMethodDescriptor(exec);
             } else {
                 md = AbstractTestBase.METHOD_GEN.generateRandomDescriptor(exec);
+                isValid = false;
```

Comment:
Is there a (remote) chance that `generateRandomDescriptor()` generates a valid descriptor? Would the compiler control test fail in that case due to a "false negative" ("expected to fail but did not fail")? Maybe a clarifying comment here would help.

Reply:
Yes, I think that could happen. I updated that line to use `md.isValid()`.